projects/1.18/assets/architects-palette/architects_palette/lang/zh_cn.json

```
@@ -251,40 +295,87 @@
   "block.architects_palette.twisted_leaves": "扭曲树叶",
   "block.architects_palette.twisted_sapling": "扭曲树苗",
   "block.architects_palette.potted_twisted_sapling": "扭曲树苗盆栽",
+  "tag.architects_palette.twisted_logs": "扭曲原木",
+
   "block.architects_palette.basalt_tiles": "玄武岩瓦",
   "block.architects_palette.basalt_tile_wall": "玄武岩瓦墙",
   "block.architects_palette.basalt_tile_stairs": "玄武岩瓦楼梯",
   "block.architects_palette.basalt_tile_slab": "玄武岩瓦台阶",
   "block.architects_palette.basalt_tile_vertical_slab": "玄武岩瓦竖直台阶",
   "block.architects_palette.chiseled_basalt_tiles": "錾制玄武岩瓦",
   "block.architects_palette.cracked_basalt_tiles": "裂纹玄武岩瓦",
+
   "block.architects_palette.sunstone": "日长石",
   "block.architects_palette.moonstone": "月长石",
+
   "block.architects_palette.molten_nether_bricks": "炽热下界砖",
   "block.architects_palette.coarse_snow": "砂雪",
+
   "block.architects_palette.heavy_end_stone_bricks": "重质末地石砖",
   "block.architects_palette.heavy_cracked_end_stone_bricks": "重质裂纹末地石砖",
+
   "block.architects_palette.redstone_cage_lantern": "红石罩灯",
   "block.architects_palette.glowstone_cage_lantern": "荧石罩灯",
   "block.architects_palette.algal_cage_lantern": "海藻罩灯",
+
   "block.architects_palette.grinning_acacia_totem": "龇牙的金合欢木图腾",
   "block.architects_palette.placid_acacia_totem": "平静的金合欢木图腾",
   "block.architects_palette.shocked_acacia_totem": "惊诧的金合欢木图腾",
   "block.architects_palette.blank_acacia_totem": "空白金合欢木图腾",
   "block.architects_palette.acacia_totem_wing": "金合欢木图腾翼",
+
   "block.architects_palette.ender_pearl_block": "末影珍珠块",
+
   "block.architects_palette.entrails_slab": "肠子台阶",
   "block.architects_palette.entrails_vertical_slab": "肠子竖直台阶",
   "block.architects_palette.entrails_stairs": "肠子楼梯",
+
   "block.architects_palette.oak_boards": "橡木宽木板",
+  "block.architects_palette.oak_board_slab": "橡木宽板台阶",
+  "block.architects_palette.oak_board_stairs": "橡木宽板楼梯",
+  "block.architects_palette.oak_board_vertical_slab": "橡木宽板竖直台阶",
+  "block.architects_palette.oak_board_wall": "橡木宽板墙",
   "block.architects_palette.dark_oak_boards": "深色橡木宽木板",
+  "block.architects_palette.dark_oak_board_slab": "深色橡木宽板台阶",
+  "block.architects_palette.dark_oak_board_stairs": "深色橡木宽板楼梯",
+  "block.architects_palette.dark_oak_board_vertical_slab": "深色橡木宽板竖直台阶",
+  "block.architects_palette.dark_oak_board_wall": "深色橡木宽板墙",
   "block.architects_palette.spruce_boards": "云杉宽木板",
+  "block.architects_palette.spruce_board_slab": "云杉木宽板台阶",
+  "block.architects_palette.spruce_board_stairs": "云杉木宽板楼梯",
+  "block.architects_palette.spruce_board_vertical_slab": "云杉木宽板竖直台阶",
+  "block.architects_palette.spruce_board_wall": "云杉木宽板墙",

```

| **审查级别** |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思 |



原文：**Spruce Board Wall**
```suggestion
  "block.architects_palette.spruce_board_wall": "云杉宽木板墙",
```

---

算是对everycomp的妥协？
改了